roles/splunk/tasks/configure_license.yml

```diff
@@ -1,4 +1,33 @@
 ---
+- block:
+  - name: Copy Licenses to Master
+    copy:
+      src: "{{ item }}"
+      dest: "/tmp/{{ item }}"
+    loop: "{{ splunk_licenses }}"
+    
+  - name: Install license on licensemaster
+    command: "{{ splunk_home }}/bin/splunk add licenses /tmp/{{ item }} -auth {{ splunk_auth }}"
+    become: true
+    become_user: "{{ splunk_nix_user }}"
+    loop: "{{ splunk_licenses }}"
+    ignore_errors: yes
```

Comment:
The play should fail if there is an error. Please remove `ignore_errors: yes`

Reply:
Yeah that would be a little bit tricky. 
If the license already exists we receive one, if the license expired we receive an error, if the license is invalid we receive one, but one of them is necessary to stop the play. As splunk can run without the license.
So the simplest way was to ignore errors, only changed is necessary, because we need to restart splunk.

Reply:
Okay, this is good then.

Reply:
I'm not a fan of ignoring errors as a means to accomplish idempotency.  Additional work should be done to make this process idempotent, whether by changing the method to use the API, or by parsing the output of the command for indication that it failed only because of a duplicate license.